roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -43,13 +54,13 @@ def tags(self):
         """
         return []
 
-    @classmethod
-    def is_active(cls, task_vars):  # pylint: disable=unused-argument
+    @staticmethod
```

Comment:
It should not be a `staticmethod` because typically if will need to access `task_vars` via `self`.

Even though it doesn't depend on `self` here, I'd still make it consistent across the class hierarchy -- easier for readers of the code.